homeassistant/helpers/translation.py

```diff
@@ -528,6 +529,34 @@ def async_translations_loaded(hass: HomeAssistant, components: set[str]) -> bool
     )
 
 
+@callback
+def async_get_exception_message(
+    translation_domain: str | None,
+    translation_key: str | None,
+    translation_placeholders: dict[str, str] | None = None,
+    language: str = "en",
+) -> str | None:
+    """Get exception message from translation cache."""
```

Comment:
We should explain the gist of the function in the first line of the docstring, maybe:
```suggestion
    """Return a translated exception message.

    Defaults to English, requires translations to already be cached.
    """
```